agentstack/_tools/agentql/__init__.py

```diff
@@ -0,0 +1,83 @@
+import os
+import httpx
+
+from typing import Optional
+
+from dotenv import load_dotenv
+load_dotenv()
```

Comment:
`load_dotenv` happens in `$ agentstack run` now so this is redundant

Reply:
removed!